agent/app/service/setting.go

```diff
@@ -103,9 +104,14 @@ func (u *SettingService) ReloadConn() error {
 		global.LOG.Errorf("update base dir failed, err: %v", err)
 		return nil
 	}
+	if err := settingRepo.Update("NodePort", fmt.Sprintf("%v", nodeInfo.NodePort)); err != nil {
+		global.LOG.Errorf("update node port failed, err: %v", err)
+		return nil
+	}
 
 	global.CONF.System.BaseDir = nodeInfo.BaseDir
 	global.CONF.System.Version = nodeInfo.Version
+	global.CONF.System.Port = fmt.Sprintf("%v", nodeInfo.NodePort)
 	global.IsMaster = nodeInfo.Scope == "master"
 	return nil
 }
```

Comment:
There are two main areas of concern when analyzing this code:

1. The use of `fmt.Errorf` instead of `return`. This could lead to more specific error messages if there is an issue with formatting the output string.

2. If a file has not been updated properly in one of the functions, it will be written to disk but without being checked. This would cause problems during subsequent checks on other files later in the process. A better approach may be writing to JSON instead and checking the content integrity after applying updates from multiple files. 

These changes should help catch inconsistencies before they occur in future builds and reduce potential errors or crashes down the line!